ChromeExtension/customDomain.html

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<!--
+ This page is shown when the extension button is clicked, because the
+ "browser_action" field in manifest.json contains the "default_popup" key with
+ value "popup.html".
+ -->
+<html>
+
+<head>
+    <title>Geniuslink Extension</title>
+    <link rel="stylesheet" href="css/main.css">
+
+
+
+    <!--
+      - JavaScript and HTML must be in separate files: see our Content Security
+      - Policy documentation[1] for details and explanation.
+      -
+      - [1]: https://developer.chrome.com/extensions/contentSecurityPolicy
+     -->
+    <script type="text/javascript" src="js/jquery.min.js"></script>
+    <script type="text/javascript" src="js/utilities.js"></script>
+    <script type="text/javascript" src="js/servicestack.js"></script>
+    <script type="text/javascript" src="js/GeniusLinkServiceClient.js"></script>
+    <script type="text/javascript" src="js/knockout.js"></script>
+    <script type="text/javascript" src="js/customDomainViewModel.js" defer="defer"></script>
+</head>
+
+<body id="body">
+    <div name="wrappedcontent" style="padding: 20px; width: 300px">
```

Comment:
For now, the inline styles are fine, but as we continue to develop this, we should try to move more styles into the stylesheet.